hooks/scene_operation_tk-shell.py

```diff
@@ -68,4 +68,7 @@ def execute(self, operation, file_path, context, parent_action, file_version, re
             return os.getcwd()
         elif operation == "reset":
             return True
-
+        elif operation == "open":
+            return QtGui.QMessageBox.question(
+                None, "", "Are you sure?", QtGui.QMessageBox.Yes | QtGui.QMessageBox.No
```

Comment:
This happens when the user clicks save in the hook for tk-shell. This is there only to make the app hooks testable out of the box with the shell engine.